apps/frontend/pages/404.tsx

```diff
@@ -0,0 +1,33 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
```

Comment:
`/styles/...` is global, don't use it
read how to use css modules
in this case, you should move the file to the same directory where 404.tsx is